ue4docker/build.py

```diff
@@ -245,7 +231,7 @@ def build():
 			prereqsArgs = ['--build-arg', 'BASEIMAGE=' + config.baseImage]
 			if config.containerPlatform == 'windows':
 				prereqsArgs = prereqsArgs + [
-					'--build-arg', 'HOST_BUILD=' + str(WindowsUtils.getWindowsBuild()),
+					'--build-arg', 'FULLBASEIMAGE=' + config.fullBaseImage,
```

Comment:
I'm not 100% happy with naming of this thing, but failed to come up with a better version. We can't use "desktop image" term because `https://mcr.microsoft.com/windows/server` isn't desktop in any way.

Reply:
`DLLSRCIMAGE` perhaps? Since we never run it, there's nothing preventing someone using an image that contains _just_ the desired DLLs extracted elsewhere, for example.

Reply:
`DLLSRCIMAGE` is good, thanks for suggestion.